app.js

```diff
@@ -0,0 +1,106 @@
+const projectImage = document.querySelector('.idcard-modal');
+const projectTitle = document.querySelector('.modal-tonic');
+
+openMenu.addEventListener('click', (() => {
+  menuLinks.style.display = 'flex';
+  menuLinks.style.visibility = 'visible';
+  hamburger.style.display = 'none';
+  maincontent.classList.add('blur');
+  logo.classList.add('blur');
+}));
+
+closeMenu.addEventListener('click', (() => {
+  menuLinks.style.display = 'none';
+  menuLinks.style.visibility = 'hidden';
+  hamburger.style.display = 'flex';
+  maincontent.classList.remove('blur');
+  logo.classList.remove('blur');
+}));
+
+for (let i = 0; i < navlinks.length; i += 1) {
+  navlinks[i].addEventListener('click', () => {
+    maincontent.classList.remove('blur');
+    logo.classList.remove('blur');
+    menuLinks.style.display = 'none';
+    menuLinks.style.visibility = 'hidden';
+    hamburger.classList.remove('blur');
+    hamburger.style.display = 'flex';
+  });
+}
+
+/// /////////cards/////////
+
+const storeData = [
+  {
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign- ups required',
+    imagesrc: 'Snapshoot Portfolio.png',
+    technologies: ['html', 'css', 'javaScript'],
+  },
+  {
+    name: 'Multi-post',
+    description: 'A daily selection of privately personalized reads; no accounts or sign- ups required',
+    imagesrc: 'desktop-2.png',
+    technologies: ['html', 'css', 'javaScript'],
+  },
+  {
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign- ups required',
+    imagesrc: 'desktop-3.png',
+    technologies: ['html', 'css', 'javaScript'],
+  },
+  {
+    name: 'Multi-post',
+    description: 'A daily selection of privately personalized reads; no accounts or sign- ups required',
+    imagesrc: 'desktop-4.png',
+    technologies: ['html', 'css', 'javaScript'],
+  },
+];
+
+function disappearButton() {
+  modal.style.display = 'none';
+  maincontent.classList.remove('blur');
+}
+
+function SeeProject() {
+  modal.style.display = 'flex';
+  modal.style.visibility = 'visible';
+  maincontent.classList.add('blur');
+  projectImage.src = storeData[0].imagesrc;
+  projectTitle.innerHTML = storeData[0].name;
+}
+
+function Seeproject2() {
+  modal.style.display = 'flex';
+  modal.style.visibility = 'visible';
+  maincontent.classList.add('blur');
+  projectImage.src = storeData[1].imagesrc;
+  projectTitle.innerHTML = storeData[1].name;
+}
+
+function Seeproject3() {
+  modal.style.display = 'flex';
+  modal.style.visibility = 'visible';
+  maincontent.classList.add('blur');
+  projectImage.src = storeData[2].imagesrc;
+  projectTitle.innerHTML = storeData[2].name;
+}
+
+function Seeproject4() {
+  modal.style.display = 'flex';
+  modal.style.visibility = 'visible';
+  maincontent.classList.add('blur');
+  projectImage.src = storeData[3].imagesrc;
+  projectTitle.innerHTML = storeData[3].name;
+}
```

Comment:
- [ ] Although you did an excellent job :+1:, it's necessary to create the works section and the popup dynamically using the data from the projects array. Currently, all the HTML code for the section is generated when the page loads. This will entail modifying the existing code by removing the pre-created HTML code for the projects list and adding it through JavaScript instead. [Here](https://github.com/olawale-o/olawale-o.me/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) you can find a good example.